lib/peerEvents.js

```diff
@@ -619,6 +618,10 @@ async function _validateGraph({worker, eventMap, remotePeerId}) {
             // FIXME: ensure error type is `DataError`
             const err = new Error('PROTOCOL VIOLATION 2');
             err.name = 'DataError';
+            logger.debug('PROTOCOL 2 VIOLATION INFO', {
```

Comment:
```suggestion
            logger.error('PROTOCOL 2 VIOLATION INFO', {
```

The logger level needs to change if this code is being merged into the main branch.

Reply:
Fixed in 938b42c260bec74c7fb5446553adaee2a5b7b638.